fenix/app/metrics.yaml

```
@@ -10818,3 +10818,108 @@ shopping.settings:
     metadata:
       tags:
         - Shopping
+fx_suggest:
+  ping_type:
+    type: string
+    description: >
+      The ping's type. In other situations might be designated by an event's
+      name or an interaction field. E.g. "fxsuggest-impression",
+      "fxsuggest-block", "fxsuggest-click".
```

```suggestion
      The ping's type. Currently always "fxsuggest-click".
```

I know Desktop has the same description, but the "might be" sentence reads a bit confusing (when I read that, I thought, "are there any other types that aren't documented there?"), and I wonder if we can take it out. Let's also not mention `click` or `block` until they're implemented 😊